representative_image/Readme.md

```diff
@@ -1,20 +1,20 @@
 # Summary
 
-This plugin extracts a representative image (i.e, featured image) from the article's summary or content if not specified in the metadata. 
+This plugin extracts a representative image (i.e, featured image) from the summary or content of an article or a page, if not specified in the metadata.
 
-The plugin also removes any images from the summary after extraction to avoid duplication. 
+The plugin also removes any images from the summary after extraction to avoid duplication.
 
-It allows the flexibility on where and how to display the featured image of an article together with its summary in a template page. For example, the article metadata can be displayed in thumbnail format, in which there is a short summary and an image. The layout of the summary and the image can be varied for aesthetical purpose. It doesn't have to depend on article's content format. 
+It allows the flexibility on where and how to display the featured image of an article together with its summary in a template page. For example, the article metadata can be displayed in thumbnail format, in which there is a short summary and an image. The layout of the summary and the image can be varied for aesthetical purpose. It doesn't have to depend on article's content format.
 
 ## Installation
 
-This plugin requires BeautifulSoup:
+This plugin requires `BeautifulSoup`:
```

Comment:
Could you use this opportunity to a `requirements.txt` in this plugin directory please ?

Reply:
Added in https://github.com/getpelican/pelican-plugins/pull/1220/commits/0d71bdb636711a62ce2058f3a86ef5e085034e8b